test/unit/init/init_test.js

```diff
@@ -58,13 +58,10 @@ suite('init', () => {
     });
 
     test('fails if an unknown generator is requested', () => {
+      // Note: Do not use a fake Yeoman environment in this test so that we get
```

Comment:
good note. This may be worth mentioning in the test description as well, since this is a big part of what we're actually testing here.